tutorials/legacy/multicore/mp104_processH1.C

```diff
@@ -30,10 +30,10 @@ static std::string tutname = "mp104_processH1: ";
 static std::string logfile = "mp104_processH1.log";
 static RedirectHandle_t gRH;
 
-std::vector<std::string> files {"http://root.cern/files/h1/dstarmb.root",
-                                "http://root.cern/files/h1/dstarp1a.root",
-                                "http://root.cern/files/h1/dstarp1b.root",
-                                "http://root.cern/files/h1/dstarp2.root"};
+std::vector<std::string> files {"root://eospublic.cern.ch//eos/root-eos/h1/dstarmb.root",
```

Comment:
Note that until the xrootd over eos issue with vector reads are solved, this is a disaster for not-at-cern cases.